### ISSUE TITLE
Can't get underlying object within dragulaModel *ngFor

### ISSUE THREAD

**Issue by Dooblr** [valor-software/ng2-dragula#479]
EDIT: **I was looking for .find()**. Here's the scenario: I have an ordered list of restaurant orders, and I want to vertical drag to reorder the items in a database, which propagates to the ui via observers.

So with the onDropModel, it doesn't seem like I can get a handle to the underlying object array from which my dragula bag (a wrapper div) was generated, only the HTML element. Seeing this, I tried generating keys in the html, but the changedetector overrides dragula html moves. Not surprised.

I have:

`onDropModel(args) {
    let [e, el, target, source] = args
}`

'<div [dragula]='"bag-orders"' [dragulaModel]='orders$ | async'>
    <div *ngFor="let order of orders$ | async | filter:term | orderBy: 'orderNumber'; let i=index">         
    </div>
  </div>'

But none of those are the object. Sorry if this is a dupe.

**Comment by Dooblr** [comment]
this is what I was looking for:

dragulaService.find('the-bag-name')

all the way at the bottom of the docs ^^

**Comment by Dooblr** [comment]
Actually, on second review, this is an issue. My models get updated, and I have access to them, however on a drop.subscribe observer hit, I'm getting the previous order of my model. I need the new order to update the order in the database.

**TL;DR: Has anyone implemented numbered reordering of objects in a database after a drop?**

**Comment by Dooblr** [comment]
question scope changed, opening separate issue
